islands/GlobalSearch.tsx

```diff
@@ -70,7 +70,7 @@ export default function GlobalSearch() {
 
   useEffect(() => {
     const keyboardHandler = (e: KeyboardEvent) => {
-      if (e.metaKey && e.key === "k") {
+      if (e.key === "/" || (e.metaKey || e.ctrlKey) && e.key === "k") {
```

Comment:
this doesnt seem right; it should be /, ctrl+k, or cmd+k. See #2289

Reply:
Yeah, metaKey is cmd. Test it https://dotland-70yx88jnb2g0.deno-staging.dev/std@0.149.0

Reply:
ah wait i misread the statement, my bad. should add that body check though

Reply:
> should add that body check though
What body check?

Reply:
from #2289, as per https://github.com/denoland/dotland/pull/2289#pullrequestreview-1048348424
```
      if (e.target !== document.body) {
        return;
      }
```

Reply:
Oh, didn't notice https://github.com/denoland/dotland/pull/2289. I'll revert the commit and we can land #2289